source/crud/write-operations/bulk.txt

```diff
@@ -488,63 +490,77 @@ the other document is inserted into the ``db.things`` collection.
 The ``MongoNamespace`` instance defines the databases and collections that 
 each write operation applies to.
 
-.. code-block:: java
+.. literalinclude:: /includes/fundamentals/code-snippets/bulk-write/ClientBulkWrite.java
+   :language: java
+   :dedent:
+   :start-after: start-insert-models
+   :end-before: end-insert-models
 
-   MongoNamespace peopleNamespace = new MongoNamespace("db", "people");
-   MongoNamespace thingsNamespace = new MongoNamespace("db", "things");
+.. _java-sync-client-bulk-write-update:
 
-   List<ClientNamespacedWriteModel> bulkOperations = new ArrayList<>();
+Update Example
+~~~~~~~~~~~~~~
 
-   bulkOperations.add(ClientNamespacedWriteModel
-      .insertOne(
-         peopleNamespace, 
-         new Document("name", "Julia Smith")
-      )
-   );
+The following example shows how to use the ``bulkWrite()`` method to update
+existing documents in the ``db.people`` and ``db.things`` collections.
 
-   bulkOperations.add(ClientNamespacedWriteModel
-      .insertOne(
-         thingsNamespace, 
-         new Document("object", "washing machine")
-      )
-   );
+.. literalinclude:: /includes/fundamentals/code-snippets/bulk-write/ClientBulkWrite.java
+   :language: java
+   :dedent:
+   :start-after: start-update-models
+   :end-before: end-update-models
 
-   ClientBulkWriteResult result = mongoClient.bulkWrite(bulkOperations);
+This example increments the value of the ``age`` field by ``1`` in the
+document that has a ``name`` value of ``"Freya Polk"`` in the ``people``
+collection. It also sets the value of the ``manufacturer`` field to
+``"Premium Technologies"`` in all document that have a ``category``
+value of ``"electronic"`` in the ``things`` collection.
 
-Replace Example
-~~~~~~~~~~~~~~~
-
-The following example shows how to use the ``bulkWrite()`` method to replace
-existing documents in the ``db.people`` and ``db.things`` collections.
+If multiple documents match the query filter specified in
+a ``ClientNamespacedUpdateOneModel`` instance, the operation updates the
+first result. You can specify a sort in a `ClientUpdateOneOptions
```

Comment:
s: use 'sort order' or similar, since 'specify a sort' initially reads like it's a sort of a thing

```suggestion
first result. You can specify a sort order in a `ClientUpdateOneOptions
```